api/CcsSso.Core.Domain/Constants/Constants.cs

```diff
@@ -46,6 +46,7 @@ public static class ErrorConstant
     public const string ErrorCannotDeleteLastOrgAdmin = "ERROR_CANNOT_DELETE_LAST_ADMIN_OF_ORGANISATION";
     public const string ErrorCannotRemoveAdminRoleGroupLastOrgAdmin = "ERROR_CANNOT_REMOVE_ADMIN_ROLE_OR_GROUP_OF_LAST_ADMIN";
     public const string ErrorInvalidGroupName = "INVALID_GROUP_NAME";
+    public const string ErrordonotallowallspecialcharacterGroupName = "ENTER_YOUR_GROUP_NAME_WITHOUT_ALL_SPECIAL_CHARACTERS";
```

Comment:
Minor one. for readability point of view can you use the variable like this please ErrorDonotAllowAllSpecialCharacterGroupName